### ISSUE TITLE
Use travis to check Documentation build.

### ISSUE THREAD

**Issue by jfdm** [idris-lang/Idris-dev#2049]
We now have sphinx based documentation. In the current pipeline for documentation generation mistakes in sphinx builds for html and pdf only appear once the docs are passed onto readthedocs. Thus mistakes in reST format will only be picked up post merge into master. 

```
Should we add sphinx build testing to travis to pick up faults pre merge?
```

**Comment by david-christiansen** [comment]
Yes, we should!

**Comment by jfdm** [comment]
Erm, I cannot see the effect of building the docs in the travis build..

**Comment by ahmadsalim** [comment]
@jfdm Oh, I saw that you had a Fixes #2049 (this issue PR) which was merged and saw it was done.

**Comment by jfdm** [comment]
Yeah, i thought that too but the travis file doesn't have it...I should have time to look at this today...

**Comment by jfdm** [comment]
@ahmadsalim Yes, it appears github is not reporting that the commit was from the closed but not merged PR #2050.

**Comment by jfdm** [comment]
I am going to close this, as I think the current situation with readthedocs and turn around times are sufficient.
